ic-test-utils/src/canister/wallet.rs

```diff
@@ -47,27 +50,33 @@ fn use_identity(account_name: impl AsRef<str>) -> Result<()> {
 
 /// Execute a command with retries.
 fn execute_command_with_retry(command: &str, args: &[&str], max_retries: u32) -> Result<Output> {
-    for retry in 0..=max_retries {
-        match execute_command(command, args) {
+    let mut attempts = 0;
+
+    loop {
+        match Command::new(command).args(args).output() {
+            Ok(output) if output.status.success() => return Ok(output),
             Ok(output) => {
-                if output.status.success() {
-                    return Ok(output);
-                } else {
+                eprintln!("Command failed with status: {:?}", output.status);
+                if attempts >= max_retries {
+                    eprintln!("Exhausted all retries");
                     return Err(Error::InvalidOrMissingAccount);
```

Comment:
Yeah, this error is not related to the execution of the command but when there's output from the execution so if it not success I guess it can return this error